From 1cb8b0b9f1187b8f8b823f8182703c89c790a7d4 Mon Sep 17 00:00:00 2001
From: Zhang Sanshan <b51434@freescale.com>
Date: Wed, 1 Apr 2015 14:36:01 +0800
Subject: [PATCH 467/602] MA-6502 change nand chip

change nand chip to MT29F64G08AFAAA.
enlarge /cache partition to 512M so that it can hold ota package.
enlarge /device partition to 32M. The min LEDS for partition is 17.

Signed-off-by: Zhang Sanshan <b51434@freescale.com>
---
 sabreauto_6q/BoardConfig.mk   | 4 ++--
 sabreauto_6q/ubi/ubinize.ini  | 4 ++--
 sabreauto_6sx/BoardConfig.mk  | 4 ++--
 sabreauto_6sx/ubi/ubinize.ini | 4 ++--
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/sabreauto_6q/BoardConfig.mk b/sabreauto_6q/BoardConfig.mk
index 89f2614..6f5ed2e 100644
--- a/sabreauto_6q/BoardConfig.mk
+++ b/sabreauto_6q/BoardConfig.mk
@@ -97,8 +97,8 @@ BOARD_KERNEL_CMDLINE := console=ttymxc3,115200 init=/init video=mxcfb0:dev=ldb,b
 ifeq ($(TARGET_USERIMAGES_USE_UBIFS),true)
 #UBI boot command line.
 UBI_ROOT_INI := device/fsl/sabreauto_6q/ubi/ubinize.ini
-TARGET_MKUBIFS_ARGS := -m 4096 -e 253952 -c 4096 -x none -F
-TARGET_UBIRAW_ARGS := -m 4096 -p 256KiB $(UBI_ROOT_INI)
+TARGET_MKUBIFS_ARGS := -m 8192 -e 1032192 -c 4096 -x none -F
+TARGET_UBIRAW_ARGS := -m 8192 -p 1024KiB $(UBI_ROOT_INI)
 
 # Note: this NAND partition table must align with MFGTool's config.
 BOARD_KERNEL_CMDLINE +=  mtdparts=gpmi-nand:64m(bootloader),16m(bootimg),16m(recovery),-(root) ubi.mtd=4
diff --git a/sabreauto_6q/ubi/ubinize.ini b/sabreauto_6q/ubi/ubinize.ini
index 04855c5..bbb69a9 100644
--- a/sabreauto_6q/ubi/ubinize.ini
+++ b/sabreauto_6q/ubi/ubinize.ini
@@ -10,7 +10,7 @@ vol_alignment=1
 [cache]
 mode=ubi
 vol_id=1
-vol_size=180MiB
+vol_size=512MiB
 vol_type=dynamic
 vol_name=cache
 vol_alignment=1
@@ -18,7 +18,7 @@ vol_alignment=1
 [device]
 mode=ubi
 vol_id=2
-vol_size=10MiB
+vol_size=32MiB
 vol_type=dynamic
 vol_name=device
 vol_aligment=1
diff --git a/sabreauto_6sx/BoardConfig.mk b/sabreauto_6sx/BoardConfig.mk
index 655bf72..06e87f4 100755
--- a/sabreauto_6sx/BoardConfig.mk
+++ b/sabreauto_6sx/BoardConfig.mk
@@ -81,8 +81,8 @@ BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=tt
 ifeq ($(TARGET_USERIMAGES_USE_UBIFS),true)
 #UBI boot command line.
 UBI_ROOT_INI := device/fsl/sabreauto_6sx/ubi/ubinize.ini
-TARGET_MKUBIFS_ARGS := -m 4096 -e 253952 -c 4096 -x none -F
-TARGET_UBIRAW_ARGS := -m 4096 -p 256KiB $(UBI_ROOT_INI)
+TARGET_MKUBIFS_ARGS := -m 8192 -e 1032192 -c 4096 -x none -F
+TARGET_UBIRAW_ARGS := -m 8192 -p 1024KiB $(UBI_ROOT_INI)
 
 # Note: this NAND partition table must align with MFGTool's config.
 BOARD_KERNEL_CMDLINE +=  mtdparts=gpmi-nand:64m(bootloader),16m(bootimg),16m(recovery),-(root) ubi.mtd=5
diff --git a/sabreauto_6sx/ubi/ubinize.ini b/sabreauto_6sx/ubi/ubinize.ini
index 60b3cf7..4619e28 100644
--- a/sabreauto_6sx/ubi/ubinize.ini
+++ b/sabreauto_6sx/ubi/ubinize.ini
@@ -10,7 +10,7 @@ vol_alignment=1
 [cache]
 mode=ubi
 vol_id=1
-vol_size=180MiB
+vol_size=512MiB
 vol_type=dynamic
 vol_name=cache
 vol_alignment=1
@@ -18,7 +18,7 @@ vol_alignment=1
 [device]
 mode=ubi
 vol_id=2
-vol_size=10MiB
+vol_size=32MiB
 vol_type=dynamic
 vol_name=device
 vol_aligment=1
-- 
1.8.0

